src/main/java/org/amahi/anywhere/adapter/ServerFilesAdapter.java

```diff
@@ -261,6 +263,15 @@ public void setOnClickListener(ServerFileClickListener mListener) {
             moreInfo = itemView.findViewById(R.id.more_info);
             moreOptions = itemView.findViewById(R.id.more_options);
             progressBar = itemView.findViewById(R.id.download_progress_bar);
+
+            setUpAnimation();
+
+        }
+
+        private void setUpAnimation() {
```

Comment:
Could you please add the ```setUpAnimation``` method in a related class and then call it from both ```ServerFilesAdapter.java``` and ```RecentFilesAdapter.java```. This will make your code more robust.

Reply:
Okay.